release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -681,6 +681,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* If you use the OVN-Kubernetes cluster network provider on a cluster provisioned on Amazon Web Services (AWS), the assignment of egress IP addresses to control plane nodes with the EgressIP feature is not supported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039656[*BZ#2039656*])
```

Comment:
The limitation applies to both OVN-Kubernetes and OpenShift SDN

Reply:
@kyrtapz, updated!